audit/projects/kubernetes-public/iam.json

```diff
@@ -147,6 +147,10 @@
     },
     {
       "members": [
+        "group:k8s-infra-aws-admins@kubernetes.io",
+        "group:k8s-infra-cluster-admins@kubernetes.io",
+        "group:k8s-infra-ii-coop@kubernetes.io",
+        "group:k8s-infra-prow-oncall@kubernetes.io",
```

Comment:
Expected, this was https://github.com/kubernetes/k8s.io/pull/2005 being deployed